linode/lke/schema_resource.go

```diff
@@ -143,8 +144,48 @@ var resourceSchema = map[string]*schema.Schema{
 					Optional:    true,
 					Computed:    true,
 				},
+				"acl": {
+					Type:        schema.TypeList,
+					Description: "Defines the ACL configuration for an LKE cluster's control plane.",
+					Optional:    true,
+					Computed:    true,
+					MaxItems:    1,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"enabled": {
+								Type:        schema.TypeBool,
+								Description: "Defines default policy. A value of true results in a default policy of DENY. A value of false results in default policy of ALLOW.",
+								Computed:    true,
+								Optional:    true,
+							},
+							"addresses": {
+								Type:        schema.TypeList,
+								Description: "A list of ip addresses to allow.",
+								Optional:    true,
+								Computed:    true,
+								Elem: &schema.Resource{
+									Schema: map[string]*schema.Schema{
+										"ipv4": {
+											Type:        schema.TypeList,
```

Comment:
Would it make sense for the IPv4 and IPv6 fields to become `TypeSet` since we can't guarantee their order?